### ISSUE TITLE
The MPC page hangs indefinitely on last wait.

### ISSUE THREAD

**Issue by axlan** [chilli-axe/mpc-autofill#28]
I'm trying to do a large (full 612 card) order. Everything seems to work OK during the upload, but on the last step:

```python
# Page over to the next step from "add text to backs"
wait(driver)
driver.execute_script("javascript:oDesign.setNextStep();")
```

The page seems to be stuck indefinitely (over an hour) with the "waiting circle" animation. I've had this happen two attempts in a row. Is this an issue for anyone else, or specific to my setup?

This is with chrome `Version 91.0.4472.124 (Official Build) (64-bit)` running on Windows 10 x64 with the v3.3 release binary.

**Comment by axlan** [comment]
The chrome debug console has the following error:

```
jquery.1.6.2.min.js?v20210008210716:17 Failed to load resource: the server responded with a status of 429 ()
send @ jquery.1.6.2.min.js?v20210008210716:17
ajax @ jquery.1.6.2.min.js?v20210008210716:17
getClientIpInfo @ pub_texteditor.js?v20210008210716:8
Currency.getLocation @ pub_texteditor.js?v20210008210716:10
Currency.loadCurrencyInfo @ pub_texteditor.js?v20210008210716:10
(anonymous) @ global_footer.js?v20210008210716:2
resolveWith @ jquery.1.6.2.min.js?v20210008210716:17
ready @ jquery.1.6.2.min.js?v20210008210716:17
B @ jquery.1.6.2.min.js?v20210008210716:17
```

**Comment by axlan** [comment]
Seeing the exact same behavior with a smaller 100 card order. The only other pertinent info I can think of is that my machine previously had selenium setup with chrome. I'm not sure why this would affect anything, but I guess it's possible an unexpected version of the chrome driver might be being used.

**Comment by fklemme** [comment]
I had the same problem. I had to close everything and start all over again but then it worked fine. I also assume it's a problem with their website.

**Comment by axlan** [comment]
Since it took about 3 hours to run, I made the following mod to the scripts to try to make it less likely to crash at this step.
1. After opening chrome. but before starting the automation I waited for user input to allow me to sign in.
2. I resumed the automation to let the script fill in the cards.
3. I stopped as soon as all the cards were set to allow me to save the order.

This issue should probably be closed, but it seems like adding some automation for saving after filling the card fronts might be useful if this issue is common.
